clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -9347,6 +9352,7 @@ ProcessDeclAttribute(Sema &S, Scope *scope, Decl *D, const ParsedAttr &AL,
   case ParsedAttr::AT_ThisCall:
   case ParsedAttr::AT_Pascal:
   case ParsedAttr::AT_RegCall:
+  //case ParsedAttr::AT_RegCall4:
```

Comment:
@mikerice1969 hi, if i uncomment here, and delete L9648-9650, regcall4 won't be added into Decl because hasDeclarator(D) is true:
```
static void handleCallConvAttr(Sema &S, Decl *D, const ParsedAttr &AL) {
  if (hasDeclarator(D)) return;
```

do you know why this happened?